.devops/nix/package.nix

```diff
@@ -58,30 +59,30 @@ let
   pnameSuffix =
     strings.optionalString (suffices != [ ])
       "-${strings.concatMapStringsSep "-" strings.toLower suffices}";
-  descriptionSuffix =
-    strings.optionalString (suffices != [ ])
-      ", accelerated with ${strings.concatStringsSep ", " suffices}";
+  descriptionSuffix = strings.optionalString (
+    suffices != [ ]
+  ) ", accelerated with ${strings.concatStringsSep ", " suffices}";
+
+  mapToPythonPackages = ps: packages: map (package: ps.${package}) packages;
```

Comment:
I think this adds complexity and conceals the original issue, which is that we should be using the `substituteInPlace` and `python3.withPackages` for this, and in fact we should separate the python stuff from the C++ derivation

Reply:
The latest series of commits addresses this separation